app/src/main/java/gov/hhs/cdc/trustedintermediary/external/hapi/HapiFhirImplementation.java

```diff
@@ -20,9 +21,16 @@ public static HapiFhirImplementation getInstance() {
 
     @Override
     public <T extends IBaseResource> T parseResource(
-            final String fhirResource, final Class<T> clazz) {
+            final String fhirResource, final Class<T> clazz) throws UnableToSendOrderException {
         IParser resourceParser = CONTEXT.newJsonParser();
-        return resourceParser.parseResource(clazz, fhirResource);
+
+        try {
+            return resourceParser.parseResource(clazz, fhirResource);
+        } catch (Exception e) {
+            throw new UnableToSendOrderException(
```

Comment:
We should discuss if we need to create a new exception (e.g. `UnableToParseRequestException`). If not, we should update the error messages in `EtorDomainRegistration.handleDemographics` and `EtorDomainRegistration.handleOrders` (e.g. from `Unable to send lab order` to `Unable to parse request or send lab order`) (CC: @halprin)

Reply:
I would recommend a new exception instead of `UnableToSendOrderException`.  `FhirParseException`?